centreon/src/Core/Infrastructure/RealTime/Repository/Service/DbServiceFactory.php

```diff
@@ -85,7 +85,7 @@ public static function createFromRecord(array $data): Service
             ->setCommandLine($data['command_line'])
             ->setIsFlapping((int) $data['flapping'] === 1)
             ->setIsAcknowledged((int) $data['acknowledged'] === 1)
-            ->setIsInDowntime((int) $data['in_downtime'] === 1)
+            ->setIsInDowntime((int) $data['in_downtime'] > 0)
```

Comment:
For My knowledge, What could be return for in downtime ? according to createTableCentstorage.sql

`in_downtime` tinyint(1) NOT NULL DEFAULT 0 COMMENT '0=false, 1=true',

if === 1 is incorrect their is other value possible ?

Reply:
You are refering to `centreon_storage.resources.in_downtime`.
here we are refering to `centreon_storage.services.scheduled_downtime_depth` 
The information we are getting out of itis the same in both case but the data of the second is the number of downtimes set for the service and therefore can be > 1.